scripts/generateAssetData/baseAssets.ts

```diff
@@ -204,7 +204,7 @@ export const arbitrumNova: AssetWithNetworkName = {
   icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
   networkIcon:
     'https://assets.coingecko.com/asset_platforms/images/93/large/AN_logomark.png?1695026131',
-  explorer: 'https://nova.arbiscan.io',
-  explorerAddressLink: 'https://nova.arbiscan.io/address/',
-  explorerTxLink: 'https://nova.arbiscan.io/tx/',
+  explorer: 'https://nova-explorer.arbitrum.io',
```

Comment:
Note, Arbitrum One doesn't have a Blockscout explorer - same goes for BSC and Avalanche, see https://docs.blockscout.com/about/projects

Reply:
yeah putting up commit to leave those

Reply:
jesus 13K changes ok